### ISSUE TITLE
"Create file from scratch" on Kubernetes pages

### ISSUE THREAD

**Issue by amisskii** [containers/podman-desktop#8832]
### Is your enhancement related to a problem? Please describe

On the `Podman Pods` page, you can no longer "Apply YAML" (and paste yaml) to a Kubernetes cluster. This ticket is to add that removed functonality from that page and add it to the `Kubernetes > Pods` page. 

### Describe the solution you'd like

On the `Kubernetes Pods` page, when you click on the "Apply YAML" button instead of opening the operating system file picker copy the behavior from the `Pods` page. Allowing users to either pick a file or paste in the monaco editor. 

<img width="2460" height="1244" alt="Image" src="https://github.com/user-attachments/assets/fdc04817-82cb-44d8-860e-5594a691e79d" />

### Additional context

This will also allow us to fix the playwright tests, because it is not possible to just the file picker, but it does work to paste a file.

While the Playwright documentation describes a method to interact with file chooser dialogs, it only works for dialogs triggered from the browser context. Unfortunately, it does not support interacting with file choosers opened by desktop applications, which blocks our e2e testing workflows.

https://playwright.dev/docs/api/class-filechooser

**Comment by deboer-tim** [comment]
Just noticed this in the sprint, but didn't see a link to #10161. The Apply YAML flow is being redesigned this sprint to handle more cases / improve usability, just want to make sure we're aware and try something in the direction the design is going to avoid throwaway code.

**Comment by benoitf** [comment]
related issue https://github.com/podman-desktop/podman-desktop/issues/12172 is still unresolved

**Comment by odockal** [comment]
Since https://github.com/podman-desktop/podman-desktop/issues/12172 is resolved now, can someone work on this issue? Until then we have not means how to test deploying kube yaml on cluster, see https://github.com/podman-desktop/podman-desktop/issues/13980.

**Comment by gastoner** [comment]
I think that apply yaml in kubernetes should now look apply yaml in providers (IMHO it should be identical but instead of apply to provider there should be apply to kubernetes + context selection) 

Also I would do the file inputs writable

**Comment by odockal** [comment]
@Firewall @rujutashinde Would it be possible to prioritize this issue into next release? As right now we have e2e tests disabled for this feature and have no other way to apply kube yaml to a cluster in k8s tests.

**Comment by Firewall** [comment]
To me it clear (mostly to me) and to check if we are all on the same page. I think the suggested idea in the original ticket is not what we want anymore. We updated the `Pods` page to only show Pods from the Podman engine, as a result allowing users to select a Kubernetes cluster to apply your YAML to on the **Pods** page doesn't make much sense anymore. 

As @gastoner suggested, we should change the flow on the **Kubernetes** `Apply YAML` button so that is behaves the same way as the flow implemented in https://github.com/podman-desktop/podman-desktop/issues/12172 except that instead it also includes the selection on what K8s cluster to deploy too.

this section from the old page: 
<img width="1064" height="235" alt="Image" src="https://github.com/user-attachments/assets/80b66e77-d6d5-4ad6-9b77-e4200a2dff45" />

Do you want me to edit the original ticket @amisskii ?

**Comment by amisskii** [comment]
@Firewall Sure, feel free to edit the issue.
Also, I think it would be great to have an option for creating k8s resources from scratch, similar to what we now have on the Podman Play YAML page. This feature is also present in the OpenShift console. It could be a separate feature request ticket.

**Comment by Firewall** [comment]
This is what @gastoner implemented in the other PR 

<img width="1230" height="622" alt="Image" src="https://github.com/user-attachments/assets/719adc40-34ab-47cf-8b3c-bc5aa6eebc8f" />

I'll leave that up to the engineers to decide if that is a seperate ticket or can be handled in this one.

**Comment by gastoner** [comment]
I think it could be two issues, one matching the style/page design to old Podman's flow 
And in another add the edit Monaco thing

**Comment by Firewall** [comment]
@gastoner @amisskii After thinking this through... does the Kubernetes cluster picker make sense? You are filtered down to a single cluster on the `Kubernetes Pods` page (The one you have selected in your Context in the statusbar). Am I forgetting something?


I also edited the ticket above, can you have a look?

**Comment by gastoner** [comment]
I think there should be probably only the context picker I guess? exactly what was removed I would just add in this ticket

**Comment by Firewall** [comment]
> I think there should be probably only the context picker I guess? exactly what was removed I would just add in this ticket

Might that not be weird? You could pick a context that is not the one you are looking at, and thus you would not see anything of the yaml (kubernetes objects) you added being created.

**Comment by gastoner** [comment]
Thats true, but it was the original functionality. Maybe I would at least pre-select the current one
